### PR TITLE
Fixes #519. PX4 and Arducopter support disarm in flight with magic number in param 2

### LINKED ISSUE #519: Emergency Stop
As discussed with @LorenzMeier in this issue https://github.com/mavlink/qgroundcontrol/issues/2669 to add the interpretation for the "magic" number for parameter 2 given by this command MAV_CMD_COMPONENT_ARM_DISARM as described here http://copter.ardupilot.com/wiki/common-mavlink-mission-command-messages-mav_cmd/#copter-2

MAV_CMD_COMPONENT_ARM_DISARM

Supported by: Copter
Disarm the motors.
Note:
This was introduced in AC3.3.
Copter

Disarm the motors.
The command supports disarming on the ground and in flight.
Note:
The motors will disarm automatically after landing.
This is not a “mission command” (it can’t be used as a type of mission waypoint).
Command parameters

```
COMMAND FIELD   MISSION PLANNER FIELD   DESCRIPTION
param1      1 to arm, 0 to disarm. This only works when the vehicle is on the ground.
param2      A value of 21196 will disarm the vehicle in flight.
param3      
param4      
param5      
param6      
param7      

```

### PR DESCRIPTION
**Pull request by pklapperich**

### CODE REVIEW

**Comment by LorenzMeier** [comment]
@julianoes Please verify.

**Comment by julianoes** [comment]
@LorenzMeier we could use the same for PX4.

There is also https://mavlink.io/en/messages/common.html#MAV_CMD_DO_FLIGHTTERMINATION but that could actually mean to set failsafe values for e.g. a parachute and not disarm as such.

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -1535,6 +1535,7 @@
       <entry value="400" name="MAV_CMD_COMPONENT_ARM_DISARM" hasLocation="false" isDestination="false">
         <description>Arms / Disarms a component</description>
         <param index="1" label="Arm" minValue="0" maxValue="1" increment="1">0: disarm, 1: arm</param>
+        <param index="2" label="Arm" minValue="0" maxValue="21196" increment="21196">21196: permit disarming during flight </param>
```
Comment (amilcarlucas) [inline message_definitions/v1.0/common.xml]: The label MUST be unique inside the message!!
And there is a stray white-space in the end of the description.

**Comment by julianoes** [comment]
@amilcarlucas please check again, I added a commit.

File: message_definitions/v1.0/common.xml [hunk 2]
```diff
@@ -1535,6 +1535,7 @@
       <entry value="400" name="MAV_CMD_COMPONENT_ARM_DISARM" hasLocation="false" isDestination="false">
         <description>Arms / Disarms a component</description>
         <param index="1" label="Arm" minValue="0" maxValue="1" increment="1">0: disarm, 1: arm</param>
+        <param index="2" label="Enforce in-air" minValue="0" maxValue="21196" increment="21196">0: only arm-disarm when landed, 21196: enforce arming/disarming even in-air (during flight)</param>
```
Comment (amilcarlucas) [inline message_definitions/v1.0/common.xml]: I am not sure if the label can have whitespaces

Comment (julianoes) [reply message_definitions/v1.0/common.xml]: Hmm :thinking:, @hamishwillee do you know?

Comment (LorenzMeier) [reply message_definitions/v1.0/common.xml]: Let’s just make it simple, call it „Enforce“

Comment (julianoes) [reply message_definitions/v1.0/common.xml]: Many param labels have whitespaces, this is fine.

Comment (julianoes) [reply message_definitions/v1.0/common.xml]: @amilcarlucas could you check again, thx.

**Comment by amilcarlucas** [comment]
Pleasee in the future do not merge like this. Use the "rebase and merge" option instead.

**Comment by julianoes** [comment]
Oh, I'm sorry. If merge is not is not our workflow here we should disable the option in the github settings for this repo.

**Comment by julianoes** [comment]
I've changed it now:

Selection_003

**Comment by amilcarlucas** [comment]
Lovely, Thanks !!!!!!!!!

**Pull request merged**